resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -224,7 +225,9 @@ cloudwatch.logs.opening_in_editor=Opening Stream {0} in editor
 cloudwatch.logs.query=Open Query Editor
```

Comment:
Pulled down this PR to poke around in, you can remove the separator between this and `View log streams`, missed that in the last PR

Reply:
The separator has been removed in the latest commit.